package.json

```diff
@@ -21,7 +21,8 @@
   },
   "main": "./dist/video-js/video.js",
   "dependencies": {
-    "videojs-swf": "4.5.3"
+    "videojs-swf": "4.5.3",
+    "vtt.js": "git+https://github.com/gkatsev/vtt.js.git#shim-build"
```

Comment:
Unfortunately, not yet. All the IE8 changes went in but the shim build stuff still need to get tweaked first.